### PR TITLE
Use GQL variables instead of string formatting for GQL queries

### PR DESCRIPTION
**Pull request by TroyDota**
Using GQL variables which avoids string format issues.

Pull request checklist:

- [ ] `CHANGELOG.md` was updated, if applicable
- [ ] Documentation in docs/ or install-docs/ was updated, if applicable

### CODE REVIEW

File: pajbot/apiwrappers/seventv.py [hunk 1]
```diff
@@ -68,18 +75,17 @@ def get_global_emotes(self, force_fetch=False):
...
+    user(id: $id) {
+        emotes {
+            id
+            name
+        }
+    }
+}"""
 
-        params = {"query": query_string}
+        params = {"query": query_string, "variables": {"id": channel_name}}
```
Comment (pajlada) [inline pajbot/apiwrappers/seventv.py:88]: can you do same formatting here as you did on line 48

**Review by pajlada — APPROVED** [review]
LGTM

**Comment by RAnders00** [comment]
I don‘t really see any problem with the original implementation, since we‘re interpolating numbers, not arbitrary strings…

**Comment by pajlada** [comment]
> I don‘t really see any problem with the original implementation, since we‘re interpolating numbers, not arbitrary strings…

Yeah safety-wise I guess we'd be fine, this seems to be more readable and standard when it comes to GQL queries that take parameters though so I'd prefer this

**Pull request merged**